libs/hscim/src/Web/Scim/Class/Group.hs

```diff
@@ -85,6 +86,8 @@ data GroupSite tag route = GroupSite
   { gsGetGroups ::
       route
         :- QueryParam "filter" Filter
+          :> QueryParam "startIndex" Natural
+          :> QueryParam "count" Natural
```

Comment:
This should not be `Natural`, the RFC specifies negative values don't result in errors but result in some defaults.